eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -68,6 +68,26 @@ jobs:
 
       - script: |
           node common/scripts/install-run-rush.js install
+        displayName: 'Install Rush'
+
+      - script: |
+          node common/scripts/install-run-rush.js lint $(rushFlags)
+        displayName: 'Run lint'
+
+      - task: CopyFiles@2
+        inputs:
+          contents: 'sdk/**/**/*lintReport.html'
+          targetFolder: $(Build.ArtifactStagingDirectory)
+          flattenFolders: true
+        displayName: 'Copy packages'
+
+      - task: PublishBuildArtifacts@1
+        condition: succeededOrFailed()
+        displayName: 'Publish lint reports'
+        inputs:
+          artifactName: reports
+
+      - script: |
           node common/scripts/install-run-rush.js audit $(GeneratedPackageTargets)
         condition: and(succeeded(), eq(variables['RunNpmAudit'], 'true'))
         displayName: "Audit packages"
```

Comment:
While you’re here you could make this consistent and call it “Audit libraries” :)

Reply:
done